models/OCSeminarSeries.php

```diff
@@ -51,4 +51,25 @@ public static function getSeries($course_id)
 
         return $return;
     }
+
+    public static function findAll()
+    {
+        return self::findBySQL("1 ORDER BY mkdate");
+    }
+
+    public static function getSeriesByUserMemberStatus($user_id, $status = 'dozent') {
```

Comment:
Why not use a sql query for this, should be faster or not?
```
SELECT series_id
FROM   oc_seminar_series
WHERE  seminar_id IN (SELECT seminar_id
                      FROM   seminar_user
                      WHERE  user_id = ?
                             AND status = ?) 
```

Reply:
Vollkommen richtig!